core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -2756,11 +2756,29 @@ private static long getMaxOfBlockAndFileSize(long blockSize, long fileSize) {
    * @param identifier
    * @param filePath
    * @param segmentId
+   * @param isTransactionalTable
    * @param isStandardTable
    * @return
    */
   public static String getBlockId(AbsoluteTableIdentifier identifier, String filePath,
       String segmentId, boolean isTransactionalTable, boolean isStandardTable) {
+    return getBlockId(identifier, filePath, segmentId, isTransactionalTable, isStandardTable,
+        false);
+  }
+
+  /**
+   * Generate the blockid as per the block path
+   *
+   * @param identifier
```

Comment:
remove @param from comment, i not explained and straight forward

Reply:
Fixed